config/config.go

```diff
@@ -154,6 +155,12 @@ type BigQueryConfig struct {
 	Dataset   string `yaml:"dataset"`
 }
 
+// CloudSourceConfig is used to configure cloudSource integration
+type CloudSourceConfig struct {
+	PrivateKeyPath    string   `yaml:"privateKeyPath"`
+	WhitelistedOwners []string `yaml:"whitelistedOwners"`
```

Comment:
Perhaps rename it to `WhitelistedProjects` since it will be a list of project ids we're dealing with, right?

Reply:
Yes. That's correct.

Related to that topic. For cloud source `repoOwner` is not really owner, is project. Should we keep as `repoOwner` for the sake of consistency?

Reply:
Yes please do keep `repoOwner`, because in other places it's used regardless of whether you're dealing with github, bitbucket or cloud source.